test/Zuora/Test/Object/ZuoraObjectTest.php

```diff
@@ -54,7 +55,7 @@ public function testdMapWrongArgumentType()
     public function testdMapWrongClassname()
     {
         $data = array('test' => array());
-        $object = new Object($data);
+        $object = new ZuoraObject($data);
         $object->map('test', 'MissingClass');
     }
 } 
```

Comment:
EOF

Reply:
Thanks! Addressed as part of `phpcs --standard=PSR2 ./test/`
Please re-visit